app/upgrades/v3/forks.go

```diff
@@ -14,32 +16,31 @@ func RunForkLogic(ctx sdk.Context, gov *govkeeper.Keeper, staking *stakingkeeper
 	FixMinCommisionRate(ctx, staking)
 }
 
-// Fixes an error where minimum deposit was set to "500 osmo"
-// This denom does not exist, which makes it impossible for a proposal to go to a vote
+// Fixes an error where minimum deposit was set to "500 osmo". This denom does
+// not exist, which makes it impossible for a proposal to go to a vote.
```

Comment:
Is there a linter your running for these, that we should add to the linter in CI? (Noticed a few period adds, and words being moved to a prior line. I imagine they were automated)

Reply:
I wrap comments at 80 chars which you can have enabled in your editor. There is also a linter for this but Im sure it would be super noisy if we enabled it (i.e. a lot of triggers on the current codebase).

Reply:
Gotcha, that makes sense! Sounds great to me, just didn't realize you were manually doing it haha